### ISSUE TITLE
Error in building docker images

### ISSUE THREAD

**Issue by sashati** [wireapp/wire-server#1698]
**Describe the bug**
When I clone the `develop` branch and run `make docker-deps docker-intermediate docker-service`, I get the below error at `docker-intermediate` state. 

```
Specified file "generic-message-proto/proto/otr.proto" for extra-source-files does not exist
wire-message-proto-lens     > configure (lib)
wire-message-proto-lens     > Configuring wire-message-proto-lens-0.1.0...

Warning: File listed in libs/wire-message-proto-lens/wire-message-proto-lens.cabal file does not exist: generic-message-proto/proto/otr.proto
wire-message-proto-lens     > build (lib)
wire-message-proto-lens     > setup: filepath wildcard 'generic-message-proto/proto/otr.proto' does not
wire-message-proto-lens     > match any files.
wire-message-proto-lens     >

Warning: File listed in libs/wire-message-proto-lens/wire-message-proto-lens.cabal file does not exist: generic-message-proto/proto/otr.proto

--  While building package wire-message-proto-lens-0.1.0 using:
      /wire-server/libs/wire-message-proto-lens/.stack-work/dist/x86_64-linux/Cabal-3.0.1.0/setup/setup --builddir=.stack-work/dist/x86_64-linux/Cabal-3.0.1.0 build lib:wire-message-proto-lens --ghc-options " -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
Progress 1/27
```
Any clue would be highly appreciated

**Comment by pcapriotti** [comment]
You need to fetch git submodules:
```
git submodule update --init --recursive
```

See https://github.com/wireapp/wire-server/blob/develop/docs/developer/dependencies.md.

**Comment by sashati** [comment]
Thanks @pcapriotti. Everything works like a charm.

**Comment by tokenwizard** [comment]
Maybe I'm missing something. I didn't see start-finish instructions of setting up with docker, but from what I have pieced together, this is what I have tried:

```
git clone https://github.com/wireapp/wire-server.git
cd wire-server
git submodule update --init --recursive
make docker-deps docker-intermediate docker-services
make: *** No rule to make target 'docker-deps'.  Stop.
```

Should I be running the make command in a different directory? Also, there is mention that there are prebuilt docker images available to download, but there are 25 images at the link given. Do we need to download all of those? Or only certain ones? 

Thanks!
